test/controllers/eventResponse.test.js

```diff
@@ -0,0 +1,77 @@
+const Botmock = require('botkit-mock');
+const {expect} = require("chai");
+const commandController = require("../../skills/projectCommand.js");
+
+describe("Bot Response", () => {
+  beforeEach(()=>{
+    this.controller = Botmock({});
+
+    this.bot = this.controller.spawn({type: 'slack'});
+    commandController(this.controller);
+  });
+  afterEach(() => {
+    
+    this.controller.shutdown();
+  });
+  it('should test for response, Link added', () => {
+    return this.bot.usersInput([
+      {
+        type: 'slash_command',
+        user: 'someUserId',
+        channel: 'someChannel',
+        messages: [
+          {
+            text: 'add jira jira.com',
+            isAssertion: true,
+            command: '/project'
+          }
+        ]
+
+      }
+    ]).then((message) => {
+      console.log(message.text)
+        expect(message.text).to.equal("Link added")
+    })
+  })
+
+  it('should test for failure', () => {
+    return this.bot.usersInput([
+      {
+        type: 'slash_command',
+        user: 'someUserId',
+        channel: 'someChannel',
+        messages: [
+          {
+            text: 'add jira jira.com',
```

Comment:
this `text` input looks like it would make for a good passing test since each of the parameters are there -- `add param param` 
I'm just curious to know which part of this test makes it fail?

Reply:
@colbyallenc It fails because the response returns "Link added" when it expects "Eggs". This test is only for responses from the bot. What do you mean in regards to testing text? That would be testing another class I assume unless you mean to spit out what was added?

Reply:
what i mean is.. 

when you write this test, in order for it to fail, it must be written to fail, meaning, when you're testing the bot, you have to act as the user and input an incorrect kind of input and `expect` for that input to fail.. 

I think here you are hardcoding the word `egg` but really, i believe that you want to give it something wrong and expect it to spit out whatever comes along with that 'wrong' input... 

When I'm reading this test, I'm not really sure what part of it your expecting to fail because the input is correct. Since it's a response from the bot.. the bot its going to execute the string `add jira jira.com` and say 'yes, good looking input... Lind Added'. You're right by saying that the response returns 'Link Added' because this test is written for a successful 'Link Added'... not a failure.. 

But maybe I'm just not understanding...

Reply:
What it looks like youre saying here is that the bot should say 'eggs' on a correct input of `add jira jira.com`

Reply:
Oh so you mean I should change the input and let it fail automatically. So change add jira jira to something else I know would fail?

Reply:
EGGGSACTLY! hahahahaha :D

Reply:
because the user may put in some incorrect input, you want to act as them, and make sure its failing correctly, and for the right reasons.

Reply:
lol got it.